packages/data-stores/src/launch/reducer.ts

```diff
@@ -109,14 +109,6 @@ const isSidebarFullscreen: Reducer< boolean, LaunchAction > = ( state = false, a
 	return state;
 };
 
-const isExperimental: Reducer< boolean, LaunchAction > = ( state = false, action ) => {
-	if ( action.type === 'ENABLE_EXPERIMENTAL' ) {
```

Comment:
Shouldn't we remove also the action creator that is dispatching this action?
https://github.com/Automattic/wp-calypso/blob/trunk/packages/data-stores/src/launch/actions.ts#L118

Reply:
Yikes, missed that one. Thanks! Removed here: https://github.com/Automattic/wp-calypso/pull/50526/commits/7a26916d9cf8094b85a7f861b8f62cecf49b3dec